src/client/api/quoteDetailsDataTypes.ts

```diff
@@ -66,3 +66,34 @@ export type HomeQuoteDetails =
   | QuoteDataDkHomeContents
 
 export type HouseQuoteDetails = QuoteDataSeHouse
+
+export type GenericQuoteData = {
```

Comment:
I noticed this in @robinandeer 's PR so I'll import it from there.

Reply:
haha yeah I know it's not the most generic to type the field that way but it's better than just using "any", right? 😅

Reply:
Long term we should not access any of this data client side!